coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -58,6 +60,8 @@ def __init__(self, language: str, docstyle: str, markers: (Iterable, str),
                                  'actually {}).'.format(length))
 
         self._metadata = metadata
+        self._top_padding = None if padding == () else int(padding[0])
+        self._bottom_padding = None if padding == () else int(padding[1])
 
```

Comment:
using separate variables it becomes easily available at bear side. `DocComment.docstyle_definition.top_padding/bottom_padding` Also they are read as strings from the file(need int). I need to get separate values in the bear part anyhow. So why not split them in the first place.